canonical-web-and-design/jaas-dashboard
# Little cleanup

## Done

Little cleanup

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check code
